string robotName
geometry_msgs/PoseStamped task